TradeItIosTicketSDK2/TradeItSDK.swift

```diff
@@ -1,9 +1,21 @@
 import UIKit
 
 @objc public class TradeItSDK: NSObject {
+    // MARK: Settable properties
+
+    public static var theme: TradeItTheme = TradeItTheme.light()
+    public static var isPortfolioEnabled = true
+    public static var isAdServiceEnabled = false
+    public static var userCountryCode: String? // CountryCode matching standard: ISO3166 alpha-2. Used for managing broker availability.
+    public static var adService: AdService = DefaultAdService()
+    public static var welcomeScreenHeadlineText: String = "Link your broker account to enable:"
+    public static var featuredBrokerLabelText: String = "SPONSORED BROKER"
+    public static var activityViewFactory: ActivityIndicatorFactory = DefaultActivityIndicatorFactory()
```

Comment:
Should I add this to the config calls?

Reply:
If we add this to the trade it screens, yes, otherwise it could be confusing ?